### ISSUE TITLE
Update inventory of content on transition

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#3884]
### Summary

**What we are after:**
_In order to help us prioritize what to get off transition next, it would be good to revisit and update the inventory of pages on there that we made in https://github.com/fecgov/fec-cms/issues/1472._

[Quick list of things to migrate](https://docs.google.com/document/d/1OuKpUdn8RCKVDGugudHf5dGpLeXLwVkEC0_mL1hmLDU/edit) 🔒 

Spreadsheet/inventory of transition pages to migrate: https://docs.google.com/spreadsheets/d/10BodftlKF--VBavaAtDJirGVGrKY8Oow-K4M0ht2Ovw/edit#gid=1102356871 🔒 

### Related issues

_List any relevant related issue(s)_

- #1472 - "Identify redirects for classic site" - in which we made a giant spreadsheet (since moved into the shared content folder) of things on transition and classic.

### Action items
- [x] Compare inventory spreadsheet to quick list to identify any pages/items that are missing
- [x] Quick list is updated and prioritized
- [x] Quick list is shared with content team and product managers so we can make informed decisions going forward about what to move next.
- [x] Any needed tickets for future work are put in.

### Completion criteria:**
- [x] Quick list of transition pages to migrate is finalized and agreed to

**Comment by dorothyyeager** [comment]
We need to prioritize this. Rather than update a giant spreadsheet, I'm making a quick list and we'll go through directory together and update as needed.

**Comment by dorothyyeager** [comment]
Did the comparison yesterday and found a few pages that should have been redirected already, so added them to our redirect spreadsheet.

I also found a few more things we still have live and that we will need to either retire, redirect, or add to the migration list. Will share out with content members Friday/Monday and from those discussions, will be able to put in the new tickets.

**Comment by dorothyyeager** [comment]
Data shared out with content team this morning. I've groomed a couple tickets after that. We have a couple of conversations we need to have with business owners; those may result in a few more tickets.

The larger spreadsheet still needs some clean-up, but I've updated throughout. It revealed a few old things that needed redirecting, so those redirects were added to #4073.

We've developed this priority list of tickets: https://docs.google.com/document/d/1liKQT4XwW4z4evfJdMMWWdzuyeQVNGmijir4kUxfDhE/edit
